morphic.js

```diff
@@ -7519,6 +7519,10 @@ MenuMorph.prototype.addLine = function (width) {
     this.items.push([0, width || 1]);
 };
 
+MenuMorph.prototype.addSubmenu = function (key, submenu) {
+	this.addItem(key, key, null, null, null, null, null, submenu);
```

Comment:
> The web is so much nicer than it used to be!

Yeah, maybe in another century or two it'll get to be as nice as the pre-Web Internet.